### DISCUSSION TITLE
MariaDB

### DISCUSSION THREAD

**Discussion by Neu3BecTHo** [Pryaxis/TShock#2672] [Q&A] [answered]
When using the latest version of MariaDB, I write this.  

as I understand it, encoding problems... but here's what to do in such a situation and, how to say, how to do it, I do not know:>

I hope for your help.

![image](https://user-images.githubusercontent.com/87427271/193468075-95881cf1-b1e3-47a4-aadd-67c2af5156ef.png)

P.S my.ini MariaDB: 
![image](https://user-images.githubusercontent.com/87427271/193468226-49c20bfe-6457-466e-b3ef-6857cb0e00e7.png)

**Comment by hakusaro** [comment, marked as answer]
This one is a strange one. You can try converting your tables or manually converting tables to the encoding utf8mb4, or try something like the instructions here: https://stackoverflow.com/questions/47566730/force-mariadb-clients-to-use-utf8mb4. In theory that’ll force every client to use that encoding set.

Nothing in your config file looks wrong to me. It just looks like you might need more options. I know we’ve had issues with MariaDB before but I don’t know what to suggest honestly.

**Reply by Neu3BecTHo** [reply]
I was a little puzzled and just rolled back the version:< But it didn't work for me on Linux and I downloaded mysql 5.7. I put it in a separate folder, so I had to pohimichit and suffer, but... it worked

Thanks for the answer, I think I'll look at it at my leisure.
